### DISCUSSION TITLE
Cannot add plots...Pop up "22".

### DISCUSSION THREAD

**Discussion by brezapour81** [Chia-Network/chia-blockchain#1192] [Q&A]
Hi, 

Ive been plotting away and noticed today that all my plots have disappeared and keep getting a popup stating "22". see screenshots.

![error](https://user-images.githubusercontent.com/73196099/110387277-48fe4480-8059-11eb-8a89-0a1fd86e0c2b.JPG)

Things I have tried:

1. Re-installed Chia 0.2.5.
2. Deleted my key and re-imported NMENONICS key.
3. Pointed to the directory where my plots are but get the above screenshot popup.

Please help.... thanks all.

**Comment by GYFNoisyboy** [comment]
Having the same issue... found nothing else on the web

**Reply by hoffmang9** [reply]
One of the directories listed in your config.yaml has a problem.

**Reply by brezapour81** [reply]
@hoffmang9 Thank it was the config.yaml file which needed updating. 

I think with the new releases, you dont actually need to change the config.yaml file to point to mainnet, it is set out of the box.

Appreciate you help!

A really small note and honestly not a big deal but can the "Network Name" 'mainnet' be changed to read "MainNet" (capitals) :)

**Reply by Krupson080** [reply]
Could you please tell me exactly how you fixed it. I have the same issue as you.

**Reply by brezapour81** [reply]
I think you might not need to change the config.yaml file. 

Which version have you installed?

Since version 1.0.1, this has been corrected which would mean you dont need to touch the config.yaml file.

@hoffmang9 maybe you could confirm?

@Krupson080  try the following:

Edit C:\Users\USERNAME\.chia\testnet\config\config.yaml and change testnet6: GENESIS_CHALLENGE: null to GENESIS_CHALLENGE: e0c271a9db2870e650a3634c58b52015c616ed90f1d47853cbbcd158df51a969

**Comment by dlgreenwell** [comment]
I was getting this when I had a network drive that had become disconnected, just like hoffmang9 suggested.
